stock_picking_deliver_to/reports/stock_picking.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="report_stock_picking" inherit_id="stock.report_delivery_document">
+            <xpath expr="//div[@name='div_sched_date']" position="after">
+                <div class="col-auto">
```

Comment:
Please hide entire structure when `deliver_to` field is empty, so certain space could be saved when not filled